src/devices/src/legacy/rtc_pl031.rs

```diff
@@ -75,16 +71,14 @@ impl RTC {
             tick_offset: utils::time::get_time_ns(utils::time::ClockType::Real) as i64,
             match_value: 0,
             load: 0,
+            // The interrupt mask is initialised as not set.
             imsc: 0,
+            // The raw interrupt is initialised as not asserted.
             ris: 0,
             interrupt_evt,
```

Comment:
does it even make sense to keep the interrupt eventfd in the code, since we are not using it anywhere?

Reply:
No it does not but I was not sure whether to remove it or not. I guess I could remove it so that it does not become confusing, right?

Reply:
sure, since it's not used at all, it's best to remove it. I see that you just did that 👍🏻